.github/workflows/build.yml

```diff
@@ -1,25 +1,38 @@
 name: "Build and test Lua project"
+
 on: [push]
 jobs:
   build:
+    strategy:
+       fail-fast: true
+       matrix:
+          lua_version: [5.1, 5.2, 5.3, 5.4]
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}-lua-${{ matrix.lua_version }}
+      cancel-in-progress: true
+    name: "Build with Lua ${{ matrix.lua_version }}"
     runs-on: ubuntu-latest
     steps:
-      - name: "Install dependencies"
-        run: |
-          sudo apt-get update
-          sudo apt install luarocks
-          # Build dependencies (will be move to rockspec with LuaRocks 3)
-          sudo luarocks install luacheck
-          sudo luarocks install luacov
-          sudo luarocks install luacov-coveralls
-          sudo luarocks install LuaUnit
-          sudo apt install shellcheck
-      - uses: actions/checkout@v2
-      - name: "Set up JDK"
-        uses: actions/setup-java@v2
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+           fetch-depth: 1
+      - name: Install Lua ${{ matrix.lua_version }}
+        uses: leafo/gh-actions-lua@v8.0.0
+        with:
+          luaVersion: ${{ matrix.lua_version }}
+      - name: Install LuaRocks
+        uses: leafo/gh-actions-luarocks@v4
```

Comment:
I think we  decided not to use predefined github actions. Is this an exception?

Reply:
Correct. This is only a temporary solution until the GH runner is updated to Ubuntu 22.04 and contains Lua 5.4. This will be done in https://github.com/exasol/error-reporting-lua/issues/19